src/export/StructureDefinitionExporter.ts

```diff
@@ -196,14 +198,15 @@ export class StructureDefinitionExporter {
     }
```

Comment:
GitHub won't let me comment on the line I really want to comment on, but a few lines above, there is:
```ts
const structDef = this.resolve(parentName);
```

The resulting `structDef` is what this structdef is built on (it's a cloned version of the parent).  So... if that `structDef` is incomplete, then that definitely could be (and almost certainly _would_ be) a problem.  I think we need a log statement there -- and it probably should be warn level because it's almost certainly a problem.

In fact, there is one unit test that I questioned the results on.  I wonder if it's not working as expected because of this situation.  In fact, I think maybe that *is* the case...  I'll update my other comment w/ more details.

Reply:
@kjmahalingam -- I still think we need a _warning_ here if the parent is in progress.